### PR TITLE
DAppNode incentive

### PR DESCRIPTION
**Pull request by pablomendezroyo**
## DAppNode incentive
The main purpose of this PR is to add a new tab for DAppNode users that are part of the incentive program promoted by Gnosis. This incentive program is to validate in Gnosis xDAI chain with 4 validators per DAppNode.

The DAppNode tab added contains the same code as in the deposit tab but with a few modifications in the header and the deposit requirements. 

Overall, the DAppNode deposit calls a [SC developed by DAppNode](https://blockscout.com/xdai/mainnet/address/0x6C68322cf55f5f025F2aebd93a28761182d077c3/contracts) that contains a whitelist of address, and under the hoods call the deposit SC of Gnosis.

All the changes implemented in this PR aims to follow the same code standards in the gnosischain/gbc-deposit-ui repository

### DAppNode deposit requirements
- The DAppNode deposit must be called with a "whitelisted" address that has not expired
- The DAppNode deposit must be called with an uploaded `deposit_data.json` with exactly 4 validators

### Other changes
- CI to deploy in gh-pages
- New icon for "wrong" status

### Screenshots
addressWhitelisted
errorexpired
errorNotWhitelisted

* You can review the UI at https://dappnode.github.io/gbc-deposit-ui/

### CODE REVIEW

**Comment by maxaleks** [comment]
@pablomendezroyo, thanks for contribution!

Could you undo the changes made for GitHub Pages? We don't need it in this app.

I also think that it would be better to use the `useDeposit` hook inside your `useDappNodeDeposit` hook instead of duplicating the code

**Review by maxaleks — CHANGES_REQUESTED** [review]
@pablomendezroyo, thanks for contribution!

Could you undo the changes made for GitHub Pages? We don't need it in this app.

I also think that it would be better to use the useDeposit hook inside your useDappNodeDeposit hook instead of duplicating the code

**Comment by pablomendezroyo** [comment]
> @pablomendezroyo, thanks for contribution!
> 
> Could you undo the changes made for GitHub Pages? We don't need it in this app.
> 
> I also think that it would be better to use the `useDeposit` hook inside your `useDappNodeDeposit` hook instead of duplicating the code

Hi there!

Thank you for the feedback, will remove the gh-pages

I agree there is duplicated code inside `useDappNodeDeposit` and some code from `useDeposit` could be recycled but I am not sure if it worth it. Notice that the `validate` and the `deposit` functions are not the same so these should be implemented and the `useDeposit` hook would become too big.

Another point in favor to keep them separated is to be able to distinguish both hooks (DAppNode and Gnosis)

I just wanted to remark these points, if you still think it is better to mix both hooks I will do it

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "gbc-deposit-ui",
+  "homepage": "https://dappnode.github.io/gbc-deposit-ui/",
```
Comment (maxaleks) [inline package.json:3]: This change is unnecessary

File: public/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <base href="%PUBLIC_URL%/">
```
Comment (maxaleks) [inline public/index.html:5]: This change is unnecessary

File: package.json (hunk 1 above)
Comment (maxaleks) [inline package.json:3]: This change is unnecessary

File: public/index.html (hunk 2 above)
Comment (pablomendezroyo) [reply public/index.html]: removed

File: package.json (hunk 1 above)
Comment (pablomendezroyo) [reply package.json]: fixed

**Comment by pablomendezroyo** [comment]
@maxaleks there is an env var for the dappnode deposit contract address, included as well in the `.env.example` file. The value is:

 `REACT_APP_DAPPNODE_DEPOSIT_CONTRACT_ADDRESS=0xe9fc19d391dfd5bfadd39c92eab8ae31ef831712`

**Comment by maxaleks** [comment]
>@maxaleks there is an env var for the dappnode deposit contract address, included as well in the .env.example file. The value is:
>
>REACT_APP_DAPPNODE_DEPOSIT_CONTRACT_ADDRESS=0xe9fc19d391dfd5bfadd39c92eab8ae31ef831712

This address is different from the one you specified in the code. If the old one is in the code, please replace it with the new one

**Comment by maxaleks** [comment]
I assume that the address in the code is correct

**Comment by pablomendezroyo** [comment]
> > @maxaleks there is an env var for the dappnode deposit contract address, included as well in the .env.example file. The value is:
> > REACT_APP_DAPPNODE_DEPOSIT_CONTRACT_ADDRESS=0xe9fc19d391dfd5bfadd39c92eab8ae31ef831712
> 
> This address is different from the one you specified in the code. If the old one is in the code, please replace it with the new one

ouch my bad! 

I got confused between the addresses

I confirm the address is 
`0x6C68322cf55f5f025F2aebd93a28761182d077c3`

**Review by maxaleks — APPROVED** [review]

**Pull request merged**